config/initializers/paperclip.rb

```diff
@@ -0,0 +1,17 @@
+Paperclip::Attachment.default_options.merge!(
+  path: ':rails_root/uploads/:class/:attachment/:id_partition/:style/:filename',
+  url: '/uploads/:class/:id/:style'
+)
+
+Paperclip.options[:content_type_mappings] = {
+  wmv: 'application/vnd.ms-asf',
+  opus: 'audio/x-opus+ogg'
+}
+
+module Paperclip
```

Comment:
Detailed comment here please explaining why this is necessary. Turning off a security feature so need to explain.